.github/workflows/webflux.yml

```diff
@@ -45,6 +45,7 @@ jobs:
         name: ${{ matrix.app-type }}
         runs-on: ${{ matrix.os }}
         if: >-
+            contains(github.event.head_commit.message, '[webflux]') &&
```

Comment:
```suggestion
            contains(github.event.head_commit.message, '[webflux]') &&
            !contains(github.event.pull_request.title, '[webflux]') &&
```

Reply:
About this, I'm not sure, as I want to trigger webflux builds only if the commit message contains [webflux]

Reply:
@pascalgrimaud : ah, you are right. 👍